reference/providers/appengine.md

```diff
@@ -6,3 +6,101 @@ sidebar:
 ---
 
 {% include toc %}
+
+If you are not familiar with App Engine or any of the terms used below, please consult
+App Engine's [reference documentation](https://cloud.google.com/appengine/docs).
+
+## Resource Mapping
+
+### Account
+A Spinnaker **account** maps to a single App Engine **application**, a top-level resource that contains 
+**services**, **versions**, and **instances**. Spinnaker authenticates itself with App Engine
```

Comment:
Your use of the bolding seems to drop off in frequency inconsistently between terms